src/Config.php

```diff
@@ -130,7 +136,12 @@ private function parseConfObject($conf_object)
         return $affinity_conf;
     }
 
-    private function get_shmem()
+    private function crossScriptShmemEnabled()
+    {
+        return in_array(php_sapi_name(), $this->supported_sapis) || getenv('FORCE_ENABLE_CROSS_SCRIPT_SHMEM');
```

Comment:
If the environment variable is set to false, the getenv condition will still evaluate as true since it is coming in as a string.

WDYT about adding the following?

```php
$enableShmem = filter_var(
    getenv('FORCE_ENABLE_CROSS_SCRIPT_SHMEM'),
    FILTER_VALIDATE_BOOLEAN
);
```

Reply:
Nice find!